app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -7,7 +7,7 @@ description: |-
 related_content:
     Improve your subject knowledge: "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
     Teacher pay and benefits: "/is-teaching-right-for-me/teacher-pay-and-benefits"
-    What qualifications do you need to be a teacher? : "/is-teaching-right-for-me/qualifications-you-need-to-teach"
+    What qualifications do you need to be a teacher? : "/train-to-be-a-teacher/qualifications-you-need-to-teach"
```

Comment:
Should you change the link text here so it is no longer a question?

Reply:
We could change it to 'qualifications you need to be a teacher' but it's a key SEO term?

Reply:
leave as is so matches H1